config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Spree::Core::Engine.routes.draw do
-  # Add your extension routes here
+  namespace :api, defaults: { format: 'json' } do
+    resource :affirm, only: [], controller: 'affirm' do
```

Comment:
- I see no good reason why people would like a different controller here yet
- There are routes needed to get the api endpoit
- the namespace question is a good one, might be better to define our own namespace for this extension to keep potential conflicts away.